auraed/src/runtime/cell_name.rs

```diff
@@ -0,0 +1,38 @@
+use std::ops::Deref;
+use validation::ValidationError;
+
+#[derive(Debug)]
+pub(crate) struct CellName(String);
+
+impl validation::ValidatedField<String> for CellName {
+    fn validate(
+        input: Option<String>,
+        field_name: &str,
+        parent_name: Option<&str>,
+    ) -> Result<Self, ValidationError> {
+        let input =
+            validation::required_not_empty(input, field_name, parent_name)?;
+
+        // TODO: what makes a valid cgroup name
+        // any valid path?
+        // do we want a restriction on length?
+        // anything else?
+        // Highly restrictive for now:
+        validation::allow_regex(
+            &input,
+            &validation::DOMAIN_NAME_LABEL_REGEX,
```

Comment:
I think is fine. I would encourage us to refuse any `/` character or anything that resembles a filesystem path. 

✅ `my-cgroup-is-awesome`
❌ `my/cgroup/is/awesome`

Mostly because of how Linux creates the cgroup directory structure in `/sys/fs/cgroup`